locales/merge-en.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Added a helper script to sync language files and ensure all the same keys appear in all languages.